src/picker/rofimoji.py

```diff
@@ -75,7 +75,7 @@ def parse_arguments(self) -> argparse.Namespace:
             nargs='*',
             metavar='ACTION',
             help='How to insert the chosen characters. More than one action may be specified in '
-                 'a space separated list (e.g. --action type copy). Options: ' +
+                 'a space separated list (e.g. "--action type copy"). Options: ' +
```

Comment:
Is this really necessary? First, I think `--action "type copy" should work as well, and look more common.

Secondly, can we get it to work without the quotes? Maybe even something like `--action type --action copy` could be a valid workaround?

Reply:
The quotes aren't necessary on the command line. I just put them there to indicate "this is text you would type on the terminal". I can remove them or change them to backticks if this is confusing.